guides/v2.2/frontend-dev-guide/layouts/xml-instructions.md

```diff
@@ -302,7 +302,7 @@ Example:
 
 To pass parameters, use the <a href="#argument"><code>&lt;argument&gt;&lt;/argument&gt;</code></a> instruction.
 
-### &lt;referenceBlock&gt; and &lt;referenceContainer&gt; {#fedg_layout_xml-instruc_ex_ref}
```

Comment:
I would not eliminate the "<" and the ">" in this title, it references a node inside layout.